cmd/jwt.go

```diff
@@ -50,29 +50,12 @@ var (
 	errMalformedAuth      = errors.New("Malformed authentication input")
 )
 
-// cachedAuthenticateNode will cache authenticateNode results for given values up to ttl.
-func cachedAuthenticateNode(ttl time.Duration) func(accessKey, secretKey, audience string) (string, error) {
-	type key struct {
-		accessKey, secretKey, audience string
-	}
-
-	cache := expirable.NewLRU[key, string](100, nil, ttl)
-	return func(accessKey, secretKey, audience string) (s string, err error) {
-		k := key{accessKey: accessKey, secretKey: secretKey, audience: audience}
-
-		var ok bool
-		s, ok = cache.Get(k)
-		if !ok {
-			s, err = authenticateNode(accessKey, secretKey, audience)
-			if err != nil {
-				return "", err
-			}
-			cache.Add(k, s)
-		}
-		return s, nil
-	}
+type cacheKey struct {
+	accessKey, secretKey, audience string
 }
 
+var cacheLRU = expirable.NewLRU[cacheKey, string](100, nil, 15*time.Second)
```

Comment:
```suggestion
var cacheLRU = expirable.NewLRU[cacheKey, string](1000, nil, 15*time.Second)
```

Since we have only one, having it be 1000 seems more reasonable.

Reply:
done